pipeline/transformer/transformer_calm/src/main/scala/uk/ac/wellcome/platform/transformer/calm/CalmTransformer.scala

```diff
@@ -196,19 +196,34 @@ object CalmTransformer
       .map(AccessStatus(_))
       .toResult
 
+  def description(record: CalmRecord): Option[String] =
+    record.getJoined("Description")
+
   def physicalDescription(record: CalmRecord): Option[String] =
     (record.getList("Extent") ++ record.getList("UserWrapped6")) match {
       case Nil  => None
       case strs => Some(strs.mkString(" "))
     }
 
+  def production(record: CalmRecord): List[ProductionEvent[Unminted]] = {
+    record.getList("Date") match {
+      case Nil => Nil
+      case list => List(ProductionEvent(
```

Comment:
nit: would be clearer to me if the variable name here was `dates` rather than naming it after the type

Reply:
This function gets the production event if there is dates. I could have two function both `production` and `dates` but feel if I don't use them outside of this context that's a useless abstraction?

Reply:
Not saying there should be another function just that  `list` isn't a great variable name, and `dates` is a better description of what it is

Reply:
Misread, thought you meant the method name.